### PR TITLE
Add expectAsync support

### PR DESCRIPTION
**Pull request by jrencz**
Jasmine has native `expectAsync`, which was not taken into consideration and had to be expressed manually in rule configuration.

With these changes `expectAsync` will be supported out of the box

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Rule                              | Recommended      | Options
 ----                              | -----------      | -------
 [expect-matcher][]                | 1,               |
 [expect-single-argument][]        | 1,               |
-[missing-expect][]                | 0, `'expect()'`  | expectation function names
```
Comment (bertBruynooghe) [inline README.md]: typo `expctAsync()` ?

Comment (jrencz) [reply README.md]: Sure, thanks

File: lib/rules/no-expect-in-setup-teardown.js [hunk 2]
```diff
@@ -24,6 +24,9 @@ module.exports = function (context) {
 
       if (insideSetup) {
         var functionName = buildName(node)
+
+        console.log('functionName', functionName)
```
Comment (jrencz) [inline lib/rules/no-expect-in-setup-teardown.js]: Reminder for myself: remove

**Pull request merged**